ODIN/chain.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "odin-mainnet-freya",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "OdinChain",
+  "chain_id": "odin-mainnet-freya",
+  "bech32_prefix": "odin",
+  "daemon_name": "odind",
+  "node_home": "$HOME/.odin",
+  "genesis": "https://github.com/ODIN-PROTOCOL/networks/blob/master/mainnets/odin-mainnet-freya/final_genesis.json",
+  "key_algos": [
+    "secp256k1"`	
+  ],
+  "slip44": 494,
```

Comment:
Any reason why you chose to use a new slip44 path?

We're currently trying to get the entire Cosmos ecosystem to standardize on 118. See presentation from @dogemos for reasons here:  https://youtu.be/oCp1Ac1cRT4?t=818

Reply:
Also, according to the SLIP 044 repo, the 494 path is already claimed by BAND Protocol.

https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
> Also, according to the SLIP 044 repo, the 494 path is already claimed by BAND Protocol.
> 
> https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Hey @sunnya97 , this might be a too long-form question, but what would be the ramifications of changing the derivation path after mainnet? Would that only necessitate existing wallets migrate?

Reply:
It would just mean that the existing wallets, when importing their mnemonic into a wallet, they need to specify the old derivation path.